python/psi-api/src/com/jetbrains/python/psi/stubs/PyClassStub.java

```diff
@@ -51,4 +52,9 @@
 
   @Nullable
   String getDocString();
+
+  @Nullable // Made nullable for compatibility
+  default List<String> getSuperClassesText() {
```

Comment:
I doubt about the nullable return value here. Does it really make sense considering that you add an explicit assert on it afterwards in `PyClassElementType#serialize()`?

Reply:
@east825 I saw that @sproshev was adding compatablity code to other stubs. My deduction is the reason for the reason for this is that the stub classes are being created by some plugins for their own uses, but these stubs are not being serialized.